packages/itwin/tree-widget/src/test/trees/models-tree/ModelsTreeFiltering.test.ts

```diff
@@ -994,13 +1051,21 @@ describe("Models tree", () => {
             modelId: model.id,
             categoryId: category.id,
           });
-          return { rootSubject, model, category, physicalElement1, physicalElement2, testElement1, testElement2 };
+          return {
+            rootSubject,
+            model,
+            category,
+            ...createFilteredGroupingNodeTestInfo({
+              groupedElements: [testElement1, testElement2],
+              modelId: model.id,
+              categoryId: category.id,
+              pathUntilGroupingNode: [rootSubject, model, category],
+              parentType: "category",
+            }),
+          };
         },
-        (x) => [
-          [x.rootSubject, x.model, x.category, x.testElement1],
-          [x.rootSubject, x.model, x.category, x.testElement2],
-        ],
-        (x) => [{ parent: { type: "category", ids: [x.category.id], modelIds: [x.model.id] }, classes: [x.testElement1.className] }],
+        (x) => x.filterPaths,
+        (x) => [x.filterTargetItem],
```

Comment:
Not a big fan of this. Now I have to scroll up to `createFilteredGroupingNodeTestInfo` to see what `filterPaths` and `filterTargetItem` means. Could we only have a function that creates a grouping node and keep the functions as they were - creating the results themselves?